rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/OperatingSystem.java

```diff
@@ -55,7 +55,7 @@ public static OperatingSystem getOperatingSystem(String desc) {
         if ("LINUX".equals(desc) || "UNIX".equals(desc)) {
             return OperatingSystem.UNIX;
         }
-        if ("WINDOWS".equals(desc)) {
+        if (desc.contains("WINDOWS")) {
```

Comment:
What if 'desc' is null?

Reply:
At this point 'desc' can't be null. A previous instruction throws an exception if 'desc' is null.